### DISCUSSION TITLE
post/calculate-blob-container-size-v2

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#62] [Announcements]
# post/calculate-blob-container-size-v2

Mark Heath's Development Blog

https://markheath.net/post/calculate-blob-container-size-v2

**Comment by francesco-geri** [comment]
Hi, I have the same needs, i.e. I need to calculate the overall size of a blob or file storage. 
Me too I'm iterating in blobs (or in folders and files for the FileStorage) using Parallel of.
And me too, like Tim Metcalfe says, I've noticed that Azure Storage Explorer do the same, but it is very very fast!
I would like to know how it works!
Maybe using batch operations?
